db/migrations/20230327063318_orders.sql

```diff
@@ -0,0 +1,19 @@
+-- migrate:up
+CREATE TABLE orders (
+      id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+      reservation_number INT NOT NULL, 
```

Comment:
```suggestion
      reservation_number VARCHAR(30) NOT NULL, 
```
주문번호도 INT 보다는 VARCHAR를 활용해주세요!